src/frontend/src/hooks/finance.hooks.ts

```diff
@@ -334,14 +334,23 @@ export const useDenyReimbursementRequest = (id: string) => {
  *
  * @param fileIds The google file ids to fetch the images for
  */
-export const useDownloadPDFOfImages = (startDate: Date, endDate: Date) => {
-  return useMutation(['reimbursement-requests'], async (formData: { fileIds: string[] }) => {
-    const promises = formData.fileIds.map((fileId) => {
-      return downloadGoogleImage(fileId);
-    });
-    const blobs = await Promise.all(promises);
-    await downloadBlobsToPdf(blobs, `receipts-${startDate.toLocaleDateString()}-${endDate.toLocaleDateString()}.pdf`);
-  });
+export const useDownloadPDFOfImages = () => {
+  return useMutation(
+    ['reimbursement-requests'],
+    async (formData: { fileIds: string[]; startDate: Date; endDate: Date; refundSource: string }) => {
```

Comment:
this looks good, but this type has enough attributes that we should separate it out and name it smth (e.g. DownloadReceiptsFormInput)